### ISSUE TITLE
Reference original Sparkup's installation instructions for Vim

### ISSUE THREAD

**Issue by mattboehm** [tristen/vim-sparkup#17]
I was taking a look at the original sparkup repo today and noticed that it now includes instructions for installing for pathogen or vundle-like plugin managers: https://github.com/rstacruz/sparkup/tree/master/vim .

It'd be nice to reference those guides for here, so that people are aware that those techniques exist as alternatives to installing your fork.

**Comment by tristen** [comment]
Yep. It's time to :city_sunset: this repo completely. Would love a readme PR.

**Comment by mattboehm** [comment]
Sure, I'll make one later today.
